py/configu/core/config_schema.py

```diff
@@ -129,8 +114,7 @@ def parse(cls, scheme: "ConfigSchema") -> Dict[str, Cfgu]:
                     error_message(
                         f"invalid key {key}",
                         error_location + [key],
-                        f"path nodes mustn't contain "
-                        f"reserved words '${key}'",
+                        f"path nodes mustn't contain " f"reserved words '${key}'",
```

Comment:
`" f"` can be removed

Reply:
true. this is because of the new line width. thanks!